src/gasman.c

```diff
@@ -1478,7 +1495,8 @@ UInt ResizeBag (
         // check that enough storage for the new bag is available
         if (SpaceBetweenPointers(EndBags, CONST_PTR_BAG(bag)) < WORDS_BAG(new_size)
               && CollectBags( new_size-old_size, 0 ) == 0 ) {
-            Panic("cannot extend the workspace any more!!!!!");
+            ReportOutOfMemory(new_size,
+                             "cannot extend the workspace any more!!!");
```

Comment:
Are the different numbers of exclamation marks in the three versions of “cannot extend the workspace any more” chosen for a deliberate reason? I can see that it could help us debug. Note that there is no test coverage (I think) for this 3 exclamation mark version.

Reply:
Yes, that's why they are there (although with `ReportOutOfMemory` we now print the line number where the warning is triggered, so this trick should be less useful.